packages/amplify-category-function/src/provider-utils/awscloudformation/utils/ensure-lambda-arn-outputs.ts

```diff
@@ -0,0 +1,30 @@
+import { AmplifyCategories, AmplifySupportedService, pathManager, readCFNTemplate, stateManager, writeCFNTemplate } from 'amplify-cli-core';
+import * as path from 'path';
+
+/**
+ * updates function cfn stack with lambda execution role arn parameter
+ */
+ export const ensureLambdaExecutionRoleOutputs = async (): Promise<void> => {
```

Comment:
Since nothing is being used from the `amplify-category-function` package, instead of exporting this helper, can it live in the `amplify-provider-awscloudformation` package? Then we don't need to change the API and we don't need to use `context.amplify.invokePluginMethod` when calling this function.